SIS_import/sis_upload.rb

```diff
@@ -104,6 +131,9 @@ def Canvas_API_PUT(url, post_params)
 
 ## make Canvas API POST call
 def Canvas_API_IMPORT(url, fileName)
+	# make sure the Canvas call is within API usage quota
+	Canvas_API_CALL_check()
+	p "Canvas API IMPORT #{url}"
 	begin
 		response = RestClient.post Addressable::URI.escape(url), {:multipart => true,
```

Comment:
Comment on why you can't use Canvas_API_POST here.  It's good to not make folks spend time trying to figure out why the generic POST should not be used.